validated/dash_apps/finished_apps/data_report.py

```diff
@@ -192,13 +206,54 @@ def clear_form(n_clicks: int):
 
 
 @app.callback(
-    Output("alert_div", "children"),
+    Output("download_csv", "data"),
+    Input("csv_button", "n_clicks"),
+    [
+        State("temporality_drop", "value"),
+        State("station_drop", "value"),
+        State("variable_drop", "value"),
+        State("date_range_picker", "start_date"),
+        State("date_range_picker", "end_date"),
+    ],
+)
+def download_csv_report(
+    n_clicks: int,
+    temporality: str,
+    station: str,
+    variable: str,
+    start_time: str,
+    end_time: str,
+):
+    if n_clicks > 0:
+        try:
+            file = get_report_data_from_db(
+                station=station,
+                variable=variable,
+                start_time=start_time,
+                end_time=end_time,
+                report_type=temporality,
+            ).to_csv(index=False)
```

Comment:
The dataframe includes the station and variable id numbers, but that's not superuseful, to be honest, since it is the same for all the rows. Just drop those two columns.
```suggestion
            ).to_csv(index=False).drop(columns=["station", "variable"])
```